components/post_markdown/post_markdown.tsx

```diff
@@ -111,6 +113,7 @@ export default class PostMarkdown extends React.PureComponent<Props> {
                 hasPluginTooltips={this.props.hasPluginTooltips}
                 imagesMetadata={this.props.post && this.props.post.metadata && this.props.post.metadata.images}
                 postId={this.props.post && this.props.post.id}
+                editedAt={this.props.post && this.props.post.edit_at}
```

Comment:
Why not `this.props.post?.edit_at`?

Reply:
good question ... I was just following the structure in the rest of the props ... changed it!